backofficeAuthorizer/src/app/s3Utils.js

```diff
@@ -1,14 +1,35 @@
 const { S3Client, GetObjectCommand } = require("@aws-sdk/client-s3");
 const yaml = require( 'js-yaml');
-const { parseTagFromOpenAPIYAML } = require( "./yamlUtils.js");
+const { arraysOverlap } = require("./utils.js");
 
-const getMethodTagsFromS3 = async (event, bucket, key) => {
+const transformPathPattern = (path, servicePath) => {
+    const string = `^/${servicePath}`
+    const regex = new RegExp(string)
+
+    return path.replaceAll(/({.*})/g,"*").replace(regex, '');
```

Comment:
Mi sembra un quantificatore "mangione" penso che in questo caso sia meglio un quantificatore "a dieta".